util/mkerr.pl

```diff
@@ -449,8 +449,8 @@ sub help
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef HEADER_${lib}ERR_H
-# define HEADER_${lib}ERR_H
+#ifndef __OPENSSL_${lib}ERR_H__
+# define __OPENSSL_${lib}ERR_H__
```

Comment:
I think you forgot to remove some underscores here...

Reply:
> I think you forgot to remove some underscores here...

Fixed.